faq.md

```diff
@@ -5,4 +5,30 @@ css_id: faq
 
 # Frequently asked questions
 
+### **What makes Uptane different from other SOTA security mechanisms?**
 
```

Comment:
```suggestion

Security problems occur due to accidental disclosures, malicious attacks, disgruntled insiders.  It is not
a matter of whether a successful attack will occur, but when.  One key feature of a security system is the
ability to securely recover from an attack.  This means that an update system must have a way to securely 
recover from a key loss or compromise.  

For example, suppose a nation-state actor steals a signing key and wants to use it to distribute software (as has
happened before) [cite Iran, etc.].  The update system must provide a way to revoke the current trusted
information even if the adversary is able to be a man-in-the-middle for future communications.  Uptane 
is designed to provide strong security in cases like these and is designed so that failures are 
compartmentalized and limited in scope.  

No other automotive update system has been designed to work in such rigorous situations or has 
received the public scrutiny of Uptane.  We follow best practice in the security community by having
wide-scale, public review.  This has been proven to be essential time and time again to ensure a 
design will hold up against attackers, especially those as strong as nation-state actors.  Furthermore,
Uptane's design is heavily influenced by the design of TUF, a widely used software update system 
with a strong track record of usability and security across millions of devices.  As a free and open
standard, with no cost to use or adopt, Uptane stands alone in the automotive update space.

```

Reply:
Might be easier to review this if it were a separate PR. The linebreaks aren't helping, either.